pt/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: news_post
+title: "Lançado Ruby 2.4.0-preview1"
+author: "naruse"
+translator: "jcserracampos"
+date: 2016-06-20 18:00:00 +0900
+lang: pt
+---
+
+Estamos satisfeitos em anunciar o lançamento do Ruby 2.4.0-preview1.
+
+Ruby 2.4.0-preview1 é o primeiro *preview* do Ruby 2.4.0.
+Este preview1 está sendo lançado antes do usual porque ele inclui tantas
+funcionalidades novas e melhorias.
+Sinta-se a vontade para
+[enviar comentários](https://bugs.ruby-lang.org/projects/ruby/wiki/HowToReport)
+já que você ainda pode mudar as funcionalidades.
+
+## [Unificação de Fixnum e Bignum em Integer](https://bugs.ruby-lang.org/issues/12005)
+
+Embora [ISO/IEC 30170:2012](http://www.iso.org/iso/iso_catalogue/catalogue_tc/catalogue_detail.htm?csnumber=59579)
+não especifique detalhes da classe Integer,
+CRuby tem duas classes visíveis de Integer: Fixnum e Bignum.
+Ruby 2.4 as unifica em Integer.
+
+## [String suporta mapeamento de caixa Unicode](https://bugs.ruby-lang.org/issues/10085)
+
+`String/Symbol#upcase/downcase/swapcase/capitalize(!)` agora lida
+com mapeamento de caixa Unicode ao invés de apenas mapeamento de caixa ASCII.
+
+## Aperfeiçoamento de perfomance
+
+Ruby 2.4 também contém as seguintes melhorias de performance incluindo
+mudanças na linguagem:
+
+### [Array#max, Array#min](https://bugs.ruby-lang.org/issues/12172)
+
+`[x, y].max` e `[x, y].min` estão otimizadas para não criarem um *array* temporário
+dentro de certas condições.
+
+### [Regexp#match?](https://bugs.ruby-lang.org/issues/8110)
+
+Adicionada `Regexp#match?`, que executa uma combinação de expressão regular sem criar
+um objeto de referência de volta e mudando '$~' para reduzir a alocação de objeto.
+
+### Outras melhorias de perfomance
+
+* [acelerada o acesso a variável de instância](https://bugs.ruby-lang.org/issues/12274)
+
+## Debugando
+
+### [Thread#report_on_exception e Thread.report_on_exception](https://bugs.ruby-lang.org/issues/6647)
+
+Ruby ignora exceções em *threads* a não ser que outra *thread* se junte explicitamente a ela.
+Com `report_on_exception = true`,
+você pode notar se uma *thread* morreu devido a uma exceção não processada.
```

Comment:
I think in this case `notar` is correct, these are false cognates

- `Notice [en]` means `notar [pt]`
- `Note [en]` means `notificar [pt]`